SokolTextGame/Commands.cs

```diff
@@ -17,11 +17,13 @@ public class Exit : ICommand
     {
         public void Execute(string[] words, World world)
         {
+            Console.ForegroundColor = ConsoleColor.Blue;
             Console.WriteLine("See you later..");
             Environment.Exit(0);
+            Console.ResetColor();
```

Comment:
Have you tested this? You're calling ResetColor() **after** the Exit call is finished. Meaning after the program has finished execution. Meaning this does nothing. Please lift this call so it's after the Console.WriteLine but before Exit.